app/serializers/api/admin/index_order_cycle_serializer.rb

```diff
@@ -61,6 +61,10 @@ def delete_path
         admin_order_cycle_path(object)
       end
 
+      def subscriptions_count
+        options[:subscriptions_counts][object.id]
```

Comment:
wouldn't this raise an exception when `@collection.blank?`?

Reply:
No, I think `options[:subscriptions_counts]` will be an empty array and this will return `nil`. And maybe it wouldn't even be called, because there is no order cycle to retrieve subscriptions for. But it would be better to return `0` instead of `nil` just in case. The method may be used differently in the future.